package.json

```diff
@@ -24,20 +26,5 @@
   "bugs": {
     "url": "https://github.com/jillix/engine-keypress/issues"
   },
-  "engine": {
```

Comment:
Seems to be ok now: https://github.com/jillix/engine-keypress/commit/7ab6f6355410cf7773af001b50a075ad3f7ed97b :balloon: